common/src/main/java/com/wynntils/services/itemfilter/filters/LevelItemFilter.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.itemfilter.filters;
+
+import static java.lang.Integer.parseInt;
+
+import com.wynntils.models.items.properties.LeveledItemProperty;
+import com.wynntils.services.itemfilter.type.ItemFilter;
+import com.wynntils.services.itemfilter.type.ItemFilterMatcher;
+import com.wynntils.utils.type.ErrorOr;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class LevelItemFilter extends ItemFilter {
+    private static final Pattern LEVEL_RANGE_PATTERN = Pattern.compile("^(\\d+)(?:-(\\d+))?$");
+
+    public LevelItemFilter() {
+        super(List.of("lvl"));
+    }
+
+    @Override
+    public ErrorOr<ItemFilterMatcher> createMatcher(String inputString) {
+        int minLevel;
+        int maxLevel;
+        Matcher matcher = LEVEL_RANGE_PATTERN.matcher(inputString);
+        if (!matcher.find()) {
+            return ErrorOr.error(getTranslation("invalidRange", inputString));
+        }
+
+        try {
+            minLevel = parseInt(matcher.group(1));
+            maxLevel = matcher.group(2) == null ? minLevel : parseInt(matcher.group(2));
```

Comment:
I think that `lvl:75` should be interpreted as `lvl:75-♾️`, not as `lvl:75-75`. I would assume that the basic use here would be "find anything above level X", so this should be easy to do. If we really want to find *exactly* level 75, then I can do `lvl:75-75`.

Reply:
I disagree with this, i think level:x should be a constant level. If I want a range, then I put a range. Perhaps allow >60 or <70

Reply:
Ok, fine. 

@Etrenak If we go this route, then we can save the extension to support `lvl:>75` to a follow up PR, if you want.

Reply:
I agree with this.